internal/pkg/aws/codestar/codestar.go

```diff
@@ -19,31 +19,33 @@ type api interface {
 }
 
 // Connection represents a client to make requests to AWS CodeStarConnections.
-type Connection struct {
+type CodeStar struct {
 	client api
 }
 
 // New creates a new CloudFormation client.
-func New(s *session.Session) *Connection {
-	return &Connection{
+func New(s *session.Session) *CodeStar {
+	return &CodeStar{
 		codestarconnections.New(s),
 	}
 }
 
 // WaitUntilStatusAvailable blocks until the connection status has been updated from `PENDING` to `AVAILABLE` or until the max attempt window expires.
-func (c *Connection) WaitUntilStatusAvailable(ctx context.Context, connectionARN string) error {
+func (c *CodeStar) WaitUntilStatusAvailable(ctx context.Context, connectionARN string) error {
```

Comment:
nit:
```suggestion
func (c *CodeStar) WaitUntilConnectionStatusAvailable(ctx context.Context, connectionARN string) error {
```
since the struct has changed names from Connection to CodeStar

Reply:
Ah, makes sense. Thanks, done.